packages/dashmate/configs/defaults/getBaseConfigFactory.js

```diff
@@ -310,7 +310,7 @@ export default function getBaseConfigFactory() {
           tenderdash: {
             mode: 'full',
             docker: {
-              image: 'dashpay/tenderdash:1.3',
+              image: 'dashpay/tenderdash:1',
```

Comment:
I still believe this should be 1.4. Using 1 can cause use of - for example - 1.0.0 (if it's already pulled) which is not compatible.

We had issues with drive where people didn't do dashmate update  (or it failed?) and were reporting issues. Using more precise tag should fix this issue, and enforce update even if the user doesn't call `dashmate update` directly.

And it will protect against unintended minor version updates.

Reply:
Will you change your opinion if dashmate will automatically pull images after updating the package version?

Reply:
This will definitely help, but will still not be perfect. The issue is that docker does not support semver :)

Accepting for now, but needs more discussion.